core/src/actions/destinations.ts

```diff
@@ -189,6 +190,8 @@ export class DestinationEdit extends AuthenticatedAction {
       destination.groupId
     ) {
       await destination.unTrackGroup();
+    } else if (params.triggerExport) {
+      await destination.exportGroupMembers();
```

Comment:
Should this be forced?

Reply:
(yes, nothing is actually exported if it's not forced)

Reply:
Yep - because it will look like all the exports have no changes otherwise.